server/client.py

```diff
@@ -155,3 +160,74 @@ def send_message_to(self, msg):
 
         # send msg
         ONLINE_USERS[to].send(PayloadMessage(msg.cmd, [self.user.name], msg.payload))
+
+    @register_cmd(CMD_CREATE_CHAT)
+    @login_required
+    def create_chat(self, msg):
+        """
+        Create new chat and send new chat id back to the client
+        """
+        chat_name = msg.payload
+        db_session = session()
```

Comment:
Let's move all db stuff(like DB CRUD operations or selecting/filtering) to models/utils.py as functions.